src/utils/config-utils.js

```diff
@@ -194,6 +194,39 @@ export async function getBuildTargets(buildSourceFolder) {
   return configuration && configuration.targets ? configuration.targets : []
 }
 
+export async function getBuildTarget(targetName) {
```

Comment:
can we add JSDoc here also?

Reply:
it's not clear what getBuildTarget is or does, might be worth explaining in the descriptoin